core/systems/graphite/leaderboard_enums.ex

```diff
@@ -0,0 +1,5 @@
+defmodule Systems.Graphite.LeaderboardEnums do
```

Comment:
Consider to split the enums in separate modules and use `Core.Enums.Base` as the base. See `Systems.Assignment.Templates` as an example. This will add the benefit of having human readable labels as gettext translations after running `mix gettext.extract -- merge`.